test/compilable/test19152.d

```diff
@@ -0,0 +1,12 @@
+// https://issues.dlang.org/show_bug.cgi?id=19152
+
+class TBasicAction
+{
+    int OnExecute()
+    {
+        auto name = __traits(getOverloads, this, "OnExecute").stringof;
+        return 0;
+    }
+}
```

Comment:
Is there an existing file this can be added to? This will reduce the total execution time of the tests.

Reply:
I couldn't find an obvious fit in `test/compilable`.  Shall I start a new file called `traits.d` to contain all future traits-related tests.

Reply:
That's fine by me.